ts/components/conversation/message/message-content/MessageContentWithStatus.tsx

```diff
@@ -122,11 +126,11 @@ export const MessageContentWithStatuses = (props: Props) => {
           messageId={messageId}
           isCorrectSide={isIncoming}
         />
-        <div>
+        <StyledMessageWithAuthor isOutgoing={!isIncoming}>
```

Comment:
Could we use isIncoming like we do for other styled components?

Reply:
ok